MVVM-C Push Scene.xctemplate/___VARIABLE_sceneIdentifier___.storyboard

```diff
@@ -22,28 +22,11 @@
                         <color key="backgroundColor" white="1" alpha="1" colorSpace="calibratedWhite"/>
                     </view>
                     <navigationItem key="navigationItem" id="fO7-ju-87H"/>
+                    <simulatedNavigationBarMetrics key="simulatedTopBarMetrics" prompted="NO"/>
```

Comment:
Hodíš ho jako initial? Vím, že ho tímto způsobem neinicializujeme, ale ať je to konzistentní s ostatními storyboardy.

Reply:
Jo, dobrá připomínka.